src/config/config_module.rs

```diff
@@ -11,6 +11,7 @@ use crate::config::Config;
 #[derive(Clone, Debug, Default, Setters)]
 pub struct ConfigModule {
     pub config: Config,
+    pub gql_requests: Arc<Vec<async_graphql::Request>>,
```

Comment:
can move this to extensions.

Reply:
I wasn't actually using this so I removed it